tests/utils.go

```diff
@@ -146,6 +155,48 @@ func createStorageClass() {
 	gomega.Expect(err).To(gomega.BeNil(), "while creating a default storageclass {%s}", scName)
 }
 
+func createVgPatternStorageClass() {
```

Comment:
As a separate task, we'll have to consolidate and make a flexible storage class creation function so that we don't write a new function for every new tweak in storage class field.

Reply:
Makes sense.